modules/telegram_module.py

```diff
@@ -69,11 +70,13 @@ def wrapper(bot, update, user_data=None):
     @staticmethod
     def get_help_text(func):
         argspec = inspect.getfullargspec(func).args[1:]
-        funcspec = '/%s %s\n' % (func.__name__,
-                                 ' '.join(['<%s>' % arg for arg in argspec]))
+        varargs = inspect.getfullargspec(func).varargs
+        funcspec = '/%s %s' % (func.__name__,
+                               ' '.join(['<%s>' % arg for arg in argspec]))
+        varspec = ' <*%s>\n' % varargs if varargs else '\n'
```

Comment:
This part was already a bit ugly. Maybe this could be done more cleanly by pulling the list comprehension  and nested if out of the string format, and do it beforehand. Format strings probably help with readability too

Reply:
Refactored the method, now has three staticmethods